composer.json

```
@@ -3,14 +3,7 @@
     "description": "Transbank SDK",
     "keywords": ["payments", "transbank", "api", "sdk"],
     "require": {
-        "php": ">=5.5.0",
-        "ext-curl": "*",
-        "ext-json": "*",
-        "ext-mbstring": "*",
-        "ext-soap": "*",
-        "ext-dom": "*",
-        "ext-mcrypt": "*",
-        "ext-openssl": "*"
+        "php": ">=5.5.0"
```

Why did you delete *all* of the extensions and not just the ones added for webpay? I think that `curl` and `json` and `mbstring` are required by onepay anyway

---

As I said, this only works to fail the install of composer library if in the host does not have those installed. For that reason i have removed all the extension here becouse if you have not the extension installed you could install lib anyway but will fail when you use it and it should say that XXXX function does not found.

---

Ok, i have put the old extensions again. Its ready to review again.